muckrock/foia/tasks.py

```diff
@@ -572,20 +574,19 @@ def parse_name(name):
         return foia_pks, file_datetime
 
     @transaction.atomic
-    def import_key(key, storage_bucket, comm, log):
+    def import_key(key, bucket, storage_bucket, comm, log):
         """Import a key"""
-
         foia = comm.foia
-        file_name = os.path.split(key.name)[1]
+        file_name = os.path.split(key)[1]
 
         # first parameter is instance, but we do not have one yet
         # luckily, it is only used if the upload_to for the field is
         # a callable, which it is not, so it is safe to pass in None
         full_file_name = FOIAFile.ffile.field.generate_filename(None, file_name)
         full_file_name = default_storage.get_available_name(full_file_name)
 
-        new_key = key.copy(storage_bucket, full_file_name)
-        new_key.set_acl("public-read")
+        new_obj = storage_bucket.Object(full_file_name)
+        new_obj.copy_from(CopySource={ 'Bucket': bucket.name, 'Key': key}, ACL="public-read")
```

Comment:
Should the ACL here be `settings.AWS_DEFAULT_ACL`? (This is unlikely to break anything for Muckrock, but it could be a bug on our end.)

Reply:
Also, we should be sure to test the auto-imports in our environment before pushing this upstream.

Reply:
will refactor to use the settings var and I'll consider it tested after this morning

Reply:
I think I actually caught and updated it to point to settings this morning, so it should be all set